docs/release-notes/next-iris/GH-3256-fix-store-map-bug.md

```diff
@@ -0,0 +1 @@
+* Fix a bug when returning maps from remote calls.
```

Comment:
This was long ago activated...
https://github.com/aeternity/aeternity/pull/3256/commits/f05ec515471345063cbc7a7523a0ec4d3dd43839

Reply:
This was related to this PR: https://github.com/aeternity/aeternity/pull/3256/commits